sda/cmd/s3inbox/proxy.go

```diff
@@ -510,7 +510,7 @@ func formatUploadFilePath(filePath string) (string, error) {
 	outPath := strings.ReplaceAll(filePath, "\\", "/")
 
 	// [\x00-\x1F\x7F] is the control character set
-	re := regexp.MustCompile(`[\\:\*\?"<>\|\x00-\x1F\x7F]`)
+	re := regexp.MustCompile(`[\\<>"\|\x00-\x1F\x7F\!\*\'\(\)\;\:\@\&\=\+\$\,\?\%\#\[\]]`)
 
 	dissallowedChars := re.FindAllString(outPath, -1)
```

Comment:
Good time to fix typo as well?

Reply:
> Good time to fix typo as well?

there was no typo, just re-arranging here the chars so that they are grouped as 1. other chars, 2.control chars, 3. reserved chars. Unless there is some typo here I missed?

Reply:
oh snap! good catch